src/Standards/Squiz/Sniffs/Commenting/VariableCommentSniff.php

```diff
@@ -40,6 +40,8 @@ public function processMemberVar(File $phpcsFile, $stackPtr)
             T_STRING       => T_STRING,
             T_NS_SEPARATOR => T_NS_SEPARATOR,
             T_NULLABLE     => T_NULLABLE,
+            T_TYPE_UNION   => T_TYPE_UNION,
+            T_TYPE_INTERSECTION   => T_TYPE_INTERSECTION,
```

Comment:
The coding standard used for this repository requires that equal signs line up vertically within a block. See also https://github.com/PHPCSStandards/PHP_CodeSniffer/issues/155#issuecomment-1867663412

Reply:
Ah... Damn! I should have been able to see this myself. Thanks a lot for your super prompt and helpful suggestion